bokeh/io.py

```diff
@@ -377,15 +375,12 @@ def save(obj, filename=None, resources=None, title=None, state=None, validate=Tr
     Returns:
         filename (str) : the filename where the HTML file is saved.
 
-    Raises:
-        RuntimeError
-
     '''
     if state is None:
         state = _state
 
     filename, resources, title = _get_save_args(state, filename, resources, title)
-    _save_helper(obj, filename, resources, title, validate)
+    _save_helper(obj, filename, resources, title)
```

Comment:
I commented that poorly, the `validate` value comes from the `bokeh.io.save` function signature, which is user facing. How do we deprecate kwargs in Bokeh?